lib/connection-parameters.js

```diff
@@ -82,6 +83,9 @@ ConnectionParameters.prototype.getLibpqConnectionString = function(cb) {
   if(this.database) {
     params.push("dbname='" + this.database + "'");
   }
+  if(this.replication) {
+    params.push("replication='" + (this.database === true ? "true" : this.replication) + "'");
```

Comment:
Oh, sorry. That's pipelinedb link..
please check this out :)

https://github.com/postgres/postgres/blob/839da3c7d9b76b178910ce2999f7cbbdf6e91f70/src/bin/pg_basebackup/streamutil.c#L110